src/_gettsim/functions_loader.py

```diff
@@ -571,6 +606,129 @@ def aggregation_func(source_col, group_id):
     return aggregation_func
 
 
+def _create_parent_child_link_functions(
```

Comment:
So we are testing all the stuff that is flagged by codecov here? I seem to remember a comment on that, but thought I'd double-check.

Reply:
We can ignore the cases where the function header is marked but nothing in the function body. Flagged lines in a function body are truly not covered.